smart_open/tests/test_asb.py

```diff
@@ -0,0 +1,564 @@
+# Copyright (C) 2019 Radim Rehurek <me@radimrehurek.com>
+#
+# This code is distributed under the terms and conditions
+# from the MIT License (MIT).
+#
+import gzip
+import io
+import logging
+import os
+import time
+import uuid
+import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+from collections import OrderedDict
+
+import smart_open
+import smart_open.constants
+
+logger = logging.getLogger(__name__)
+
+try:
```

Comment:
try-catch not required.

It is reasonable to expect all dependencies to be available when running tests, right?